tesseract_core/sdk/tesseract.py

```diff
@@ -62,49 +63,77 @@ class Tesseract:
 
     def __init__(self, url: str) -> None:
         self._spawn_config = None
-        self._project_id = None
-        self._container_id = None
+        self._serve_context = None
         self._lastlog = None
         self._client = HTTPClient(url)
 
+    @classmethod
+    def from_url(cls, url: str) -> Tesseract:
+        obj = cls.__new__(cls)
+        obj.__init__(url)
+        return obj
+
     @classmethod
     def from_image(
         cls,
         image: str,
         *,
         volumes: list[str] | None = None,
         gpus: list[str] | None = None,
-    ):
+    ) -> Tesseract:
```

Comment:
Uh, weird, I thought our static type checks would catch this, at least in the SDK. Will check that

Reply:
Yep those were broken across the whole codebase 🤦

Reply:
Created an issue